src/commands/infra/generate.ts

```diff
@@ -175,3 +175,95 @@ export const validateRemoteSource = async (
   }
   return true;
 };
+
+/**
+ * Creates "generated" directory if it does not already exists
+ *
+ * @param projectPath Path to the definition.json file
+ */
+export const createGenerated = async (projectPath: string) => {
+  const newGeneratedPath = projectPath + "-generated";
+  mkdirp(newGeneratedPath, err => {
```

Comment:
This looks like a function with a callback - is it ok that the directory won't yet exist potentially when createGenerated returns?

If not, consider using promisify (https://2ality.com/2017/05/util-promisify.html) to convert mkdirp into a promisified function and `await` its results.  (or mkdirp.sync which I see also exists earlier in the file)